src/general/MainMenu.cs

```diff
@@ -266,11 +266,17 @@ private void SwitchMenu()
 
     private void CheckModFailures()
     {
-        var errors = ModLoader.Instance.GetAndClearModErrors();
+        var errors = ModLoader.Instance.GetModErrors();
 
         if (errors.Count > 0)
         {
-            modLoadFailures.ExceptionInfo = string.Join("\n", errors);
```

Comment:
Wouldn't just changing the errors on this line to `errors.Select(e => e.ErrorMessage)` work instead of having to add 4 lines of new code here?

Reply:
Yeah it would, I just did not know that.